From 829966816cad14fd18f9238c4603f18105daacdc Mon Sep 17 00:00:00 2001
From: Guillem Jover <gjover@sipwise.com>
Date: Tue, 26 Jan 2021 05:10:18 +0100
Subject: [PATCH] Dh_Buildsystems: Add golang as a third-party build system
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The golang buildsystem has now gained auto-detection support for
v2+ modules in git, when they contain a «go.mod» file. Adding it
to the third-party list makes it possible to remove, in many
cases, the explicit --buildsystem argument to dh.

Signed-off-by: Guillem Jover <gjover@sipwise.com>
---
 lib/Debian/Debhelper/Dh_Buildsystems.pm | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Debian/Debhelper/Dh_Buildsystems.pm b/lib/Debian/Debhelper/Dh_Buildsystems.pm
index 08b14777..849a6d56 100644
--- a/lib/Debian/Debhelper/Dh_Buildsystems.pm
+++ b/lib/Debian/Debhelper/Dh_Buildsystems.pm
@@ -38,6 +38,7 @@ our @BUILDSYSTEMS = (
 our @THIRD_PARTY_BUILDSYSTEMS = (
 	'maven',
 	'gradle',
+	'golang',
 );
 
 my $opt_buildsys;
-- 
2.30.0

